src/pages/404.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Container, Link } from "@mui/material";
+
+const Page404 = () => (
+  <Box
+    sx={{
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center",
+    }}
+  >
+    <Container maxWidth="xl">
+      <Box>
+        <h1>Page doesn't exist</h1>
+      </Box>
+      <Box>
+        <Link href="/" underline="hover">

```

Comment:
please use constant with path instead string.
It's exactly what i've talked about =)